Sharma_Exercise08.R

```diff
@@ -0,0 +1,79 @@
+#######4 Nov 2022 Friday Tutorial############
+#Why we need  to subset data? If we have more data set, and we are just interested
+#on some of them. We need to subset on that condition 
+
+name <- c("Puja","Bindu","Bina","Pukar","Sujana")
+matrix <- matrix(1:4,nrow=2,ncol=2)
+list(name, matrix)
+fourthandfivthname <- name[c(4,5)]
+matrix[1,2]
+wages <- read.csv(file = "wages.csv")
+wages[15,]
+min(wages$wage)
+femaleWages <- wages[wages$gender=="female",]
+write.table(femaleWages, file="femaleWages", sep=",")
+
+Tweleveormoreyears <- wages[wages$yearsSchool>=12,c(1,4)]
+write.table(Tweleveormoreyears, file="Tweleveormoreyears", sep=",")
+TweleveYears <- wages[wages$yearsSchool==12,]
+min(TweleveYears$wage)
+#Minimum wages of 12 years of school is 0.102 
+SixteenYears <- wages[wages$yearsSchool==16,]
+min(SixteenYears$wage)
+#Minimum wages of 16 years of school is 4.18
+################################################################################
+##############################Exercise 08#######################################
+#1. 
+#Write R code that replicates the functionality of the head function we used in bash. 
+#Your code should define a variable with the file to return lines from and a variable 
+#representing the number of lines to be returned from the top of the indicated file. 
+#The selected file content should be printed to the terminal in R. 
+#Do not use the head function in R. Be sure you are using square bracket 
+#indexing to subset the tabular file.
+
+#set working directory
+setwd("Users/myFile")
+#read csv file of from my directory 
+data <- read.csv(file="data", header = TRUE, sep = ",", stringsAsFactors = F)
+#I will be choosing the first 10 line of the data
+#specifiy variable
+Variable <- 10 #the lines of number I want
+headData <- data[1:variable,]#first 10 row of the data and all column
```

Comment:
should be headData <- data[1:Variable,]
-0.25

Reply:
Hi quan, 
We are not running the code, so I didnt focus on the upper and lower case. 
I thought Variable/variable works, just to specify the name.

Reply:
In R variable names are case sensitive. Your code will not work if you ran it